### DISCUSSION TITLE
Como você começa um novo projeto?

### DISCUSSION THREAD

**Discussion by kvnol** [frontendbr/forum#1434] [Discussão]
Quando trabalhamos com prazos (95% das vezes) temos que buscar sermos produtivos no setup de nossos projetos, para não nos preocuparmos sempre em lembrar quais são os plugins que sempre utilizamos ou criar as tasks nos task runners.

Como vocês costumam iniciar diferentes tipos de projetos (WordPress, landing pages, sites, SPA, PWA, etc)?

Conta aí se você ainda tem problema com isso e como resolveu, ajuda os parça.

**Comment by willycamargo** [comment]
Sempre procuro algum boilerplate que me ajude com isso. Exemplos:

- [Gatsby](https://gatsbyjs.org)
- [Static Boilerplate](http://staticsiteboilerplate.com)
- [Create React App](https://github.com/facebook/create-react-app)
- [Kratos Boilerplate](https://github.com/felipefialho/kratos-boilerplate)

Quando tenho alguma necessidade específica, subo uma stack própria, mas sempre tenho em mente que isso é um processo demorado e sem "resultado visual" para mostrar para chefes e clientes.

**Comment by felipefialho** [comment]
@willycamargo Fico feliz que você usa o Kratos, haha. Me avisa qualquer melhoria ou ajuste que poderia ser feito.

_____

Em geral:

- Angular [angular-cli](https://cli.angular.io/)
- React [Create React App](https://github.com/facebook/create-react-app)
- Static Projects [Kratos Boilerplate](https://github.com/felipefialho/kratos-boilerplate)

**Comment by kazzkiq** [comment]
Estou usando o **Svelte** pra praticamente todos os novos projetos que inicio (inclusive [escrevi um post](https://claudioholanda.ch/blog/svelte-a-era-dos-frameworks-compiladores/) introdutório sobre ele recentemente), então no caso de SPAs eu começo com o [sveltejs/template](https://github.com/sveltejs/template).

No caso de sites estáticos eu tenho usado o [Eleventy](https://www.11ty.io/), mas confesso que fui dar uma olhada no [Kratos](https://github.com/felipefialho/kratos-boilerplate) e me despertou interesse, principalmente por já ter suporte PWA e usar SASS *out-of-the-box*!

Então seria:

- SPA: [sveltejs/template](https://github.com/sveltejs/template) + router
- Estáticos: [Eleventy](https://www.11ty.io/) (por hora)
- Pra criar libs JS open-source: [Open-source Starter Kit](https://github.com/kazzkiq/open-source-starter-kit)

**Comment by adamfoerster** [comment]
Também tenho usado o sveltejs/template e recomendo.

**Comment by didiraja** [comment]
Só comentando que aqui na Stone usamos Svelte e o v3 tá maravilhoso https://svelte.dev/blog/svelte-3-rethinking-reactivity
